code/modules/mob/living/carbon/human/species.dm

```diff
@@ -164,11 +164,16 @@
 
 	if(exotic_bloodtype && C.dna.blood_type != exotic_bloodtype)
 		C.dna.blood_type = exotic_bloodtype
-
+	if(C.dna.features["legs"] == "Digitigrade Legs")
+		specflags += DIGITIGRADE
+	if(DIGITIGRADE in specflags)
+		C.Digitigrade_Leg_Swap(0)
 
 /datum/species/proc/on_species_loss(mob/living/carbon/C)
 	if(C.dna.species.exotic_bloodtype)
 		C.dna.blood_type = random_blood_type()
+	if(DIGITIGRADE in specflags)
+		C.Digitigrade_Leg_Swap(1)
```

Comment:
Doesn't really add any context given FALSE would be "give them digitigrade legs if they have normal legs" and TRUE would be "give them normal legs if they have digitigrade legs"